app/controllers/BlogHeaderController.scala

```diff
@@ -0,0 +1,11 @@
+package controllers
+
+import controllers.auth.OptionallyAuthorizedAction
+import controllers.auth.Authorities.anyUser
+
+object BlogHeaderController extends Controller {
+  def show() = OptionallyAuthorizedAction(anyUser) { implicit request =>
+    Ok(views.html.components.navbar(None, None))
```

Comment:
Good first shot. A couple of improvements:

1. This isn't valid HTML, because it doesn't include a doctype or `<html>` element. You'll want to create a top-level View that just includes the navbar.
2. You can and should pass `request.user` to the View, so logged-in users appear logged in.